pkgs/development/web/postman/linux.nix

```diff
@@ -119,7 +124,7 @@ stdenv.mkDerivation rec {
     xorg.libxshmfence
   ];
 
-  nativeBuildInputs = [ wrapGAppsHook copyDesktopItems ];
+  nativeBuildInputs = [ wrapGAppsHook copyDesktopItems  ];
```

Comment:
Unrelated and unwanted whitespace change.

Reply:
Reverted.